diff --git a/nimble/host/src/ble_gattc.c b/nimble/host/src/ble_gattc.c
index aaf91354..5cc481af 100644
--- a/nimble/host/src/ble_gattc.c
+++ b/nimble/host/src/ble_gattc.c
@@ -626,7 +626,7 @@ ble_gattc_dbg_assert_proc_not_inserted(struct ble_gattc_proc *proc)
 static void
 ble_gattc_log_proc_init(const char *name)
 {
-    BLE_HS_LOG(INFO, "GATT procedure initiated: %s", name);
+    //BLE_HS_LOG(INFO, "GATT procedure initiated: %s", name);
 }
 
 #if MYNEWT_VAL(BLE_GATTC)
@@ -775,7 +775,7 @@ static void
 ble_gattc_log_notify(uint16_t att_handle)
 {
     ble_gattc_log_proc_init("notify; ");
-    BLE_HS_LOG(INFO, "att_handle=%d\n", att_handle);
+    //BLE_HS_LOG(INFO, "att_handle=%d\n", att_handle);
 }
 
 static void
